### ISSUE TITLE
color blind optimizations

### ISSUE THREAD

**Issue by KZdavid** [dysath/seat-fitting#82]
Doctrine Report is a good function, but displaying Hull and Fit with red/green is not a good idea. Suggesting to displaying with differences both in color and shape. e.g. red with hollow/green with solid. That will be a lot help. Thanks!
![77X})3BJ%$NN__J AL4O2)T](https://user-images.githubusercontent.com/11251965/135730835-a97ad733-e9ee-49b5-b870-ca81e0f195b0.png)

**Comment by Crypta-Eve** [comment]
Good point! I will have a look at options here as how to better signal this :)
